src/com/google/enterprise/adaptor/DocumentHandler.java

```diff
@@ -687,6 +690,12 @@ public OutputStream getOutputStream() throws IOException {
             ex.getResponseBody()));
         os = countingOs;
       }
+      if (null != contentTransformer) {
```

Comment:
This does have some threading issues, as we will be handling multiple requests at the same time. We're going to need a way to create multiple of the content transformers. Probably create a new one for each request.